tests/cases/fourslash/inlineVariableParensPropertyAccess.ts

```diff
@@ -1,27 +1,27 @@
-/// <reference path="fourslash.ts" />
-
-////const /*a1*/x/*b1*/ = 1 + 2;
-////const y = x * 3;
-////const /*a2*/f/*b2*/ = () => { };
-////f();
-
-goTo.select("a1", "b1");
-verify.refactorAvailable("Inline variable");
-edit.applyRefactor({
-    refactorName: "Inline variable",
-    actionName: "Inline variable",
-    actionDescription: "Inline variable",
-    newContent: `const y = (1 + 2) * 3;
-const f = () => { };
-f();`
-});
-
-goTo.select("a2", "b2");
-verify.refactorAvailable("Inline variable");
-edit.applyRefactor({
-    refactorName: "Inline variable",
-    actionName: "Inline variable",
-    actionDescription: "Inline variable",
-    newContent: `const y = (1 + 2) * 3;
-(() => { })();`
-});
\ No newline at end of file
+/// <reference path="fourslash.ts" />
+
+////const /*a1*/foo/*b1*/ = {};
+////foo.toString();
+////const /*a2*/bar/*b2*/ = 0;
+////bar.toFixed().toString();
```

Comment:
Can you add a test case for when the number already has a `.` in it:

```suggestion
////const /*a3*/baz/*b3*/ = 3.141592;
////baz.toPrecision(3);
```

Also, a test for other literals like strings and booleans?